ulb.py

```diff
@@ -12,6 +12,12 @@
 import re
 import os
 import signal
+from enum import Enum
+import logging
+
+# Define pythons log level
+logLevelNames = ["CRITICAL","ERROR","WARNING","INFO","DEBUG","TRACE"]
+logging.addLevelName(5, "TRACE") # add TRACE level
```

Comment:
About the log levels, on AirVantage (and Loggly) we only use Error, Warning, Info and Debug. I would suggest doing the same. Debug and Trace are basically the same thing.
On MQTTFE, we used to differentiate Debug and Trace but we ended up merging them because it was too impractical.

Reply:
DEBUG and TRACE is mainly for development time, not really for production.

I need 2 different log levels : one to track NAT, another for full tracing.

Here the explanation : 
CRITICAL => should never happened,
ERROR => unexpected state,
WARNING => unwanted traffic (invalid packet),
INFO => not supported traffic (fragmented ip, ip option used),
DEBUG => all about NAT,
TRACE => all output we can.